Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -0,0 +1,53 @@
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol CommentRepliesViewModelInputs {
+  /// Call with the comment/project that we are viewing replies for. Both can be provided to minimize
```

Comment:
For my own information, how is supplying both of these arguments optimizing the number of network requests?

Reply:
I just took this comment from the `CommentsViewModel` `configureWith` function. I'm assuming at some point we'll need to load the replies using the `commentRepliesQuery`. Now that I look at that query I see that it requires no `Project` info, so I will remove references to `Project` in this VM. We can add them back in later if they are necessary.